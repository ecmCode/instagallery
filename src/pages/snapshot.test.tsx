import { render } from "@testing-library/react";
import Home from "./";

describe("Home", () => {
  it("matches the snapshot", () => {
    const { asFragment } = render(<Home photos={[]} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
