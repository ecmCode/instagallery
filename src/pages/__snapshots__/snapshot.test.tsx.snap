// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col gap-10 items-center justify-center m-4"
  >
    <h1>
      Instagallery
    </h1>
    <h3>
      Search your favorite pics here
    </h3>
    <form
      action="submit"
    >
      <div
        class="searchBar"
      >
        <input
          class="input"
          type="text"
          value=""
        />
        <button
          class="btn"
          type="submit"
        >
          Search
        </button>
      </div>
    </form>
    <div />
    <div>
      <p>
        Enter at least 3 characters to get results.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
